lib/msal-browser/src/interaction_client/PopupClient.ts

```diff
@@ -540,16 +578,16 @@ export class PopupClient extends StandardInteractionClient {
                     return;
                 }
 
-                let href: string = Constants.EMPTY_STRING;
-                let hash: string = Constants.EMPTY_STRING;
+                const href = popupWindow.location.href;
```

Comment:
The popupWindow.location.href call needs to stay inside the try because this will not succeed when the popup is not same origin